form.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+const form = document.getElementById("contact");
+const email = document.getElementById("email");
+
+function isEmail(email) {
+  return email.match(
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/);
+}
+
+function checkInputs() {
+  const emailValue = email.value.trim();
+
+  function setErrorFor(input, message) {
+    const formControl = input.parentElement;
+    const small = formControl.querySelector("small");
+    formControl.className = "form-control error";
+    small.innerText = message;
+  }
+
+  if (emailValue === "") {
+    setErrorFor(email, "Email cannot be blank");
+  } else if (!isEmail(emailValue)) {
+    setErrorFor(email, "Not a valid email");
+  }
+}
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+
+  checkInputs();
+});
```

Comment:
There is an issue with the form that is causing it not to submit even though all the values are correct 👀 Please make sure that you fix this issue so that the user can send data into Formspree successfully 👌